crates/runc/src/io.rs

```diff
@@ -261,9 +239,82 @@ impl Io for NullIo {
         Ok(())
     }
 
-    /// closing only write side (should be stdout/err "from" runc process)
     fn close_after_start(&self) {
         let mut m = self.dev_null.lock().unwrap();
         let _ = m.take();
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    #[cfg(not(target_os = "macos"))]
+    use std::io::{Read, Write};
+
+    #[test]
+    fn test_io_option() {
+        let opts = IOOption {
+            open_stdin: false,
+            open_stdout: false,
+            open_stderr: false,
+        };
+        let io = PipedIo::new(1000, 1000, &opts).unwrap();
+
+        assert!(io.stdin().is_none());
+        assert!(io.stdout().is_none());
+        assert!(io.stderr().is_none());
+    }
+
+    #[cfg(not(target_os = "macos"))]
```

Comment:
I don't think it makes too much sense to target non Linux platforms, since runc is Linux specific. Lets just do 

```rust
#[cfg(test)]
#[cfg(target_os = "linux")]
mod tests {
```

Reply:
fixed